cmd/influxd/config.go

```diff
@@ -30,6 +30,12 @@ const (
 
 	// DefaultHTTPAPIPort represents the default port the HTTP API runs on.
 	DefaultHTTPAPIPort = 8086
+
+	// DefaultGraphitePort represents the default Graphite (Carbon) plaintext port.
+	DefaultGraphitePort = 2003
+
+	// DefaultGraphiteNameSeparator represents the default Graphite field separator.
+	DefaultGraphiteNameSeparator = "."
```

Comment:
To be clear, are you then suggesting we import the `graphite` package into the the config code, so I can still get at the magic numbers?